src/components/CorporateCampaign/CampaignJoinTeamForm.vue

```diff
@@ -203,6 +204,12 @@ export default {
 <style lang="scss" scoped>
 @import 'settings';
 
+.join-team-lightbox {
```

Comment:
To avoid v-deeping and overriding the lightbox container, another approach is to put the class on the contents rather than the lightbox el. 

```
<kv-lightbox>
  <div class="join-team-lightbox">lightbox contents...
</kv-lightbox>

.join-team-lightbox { max-width: 30rem; }
```

Reply:
oh good call, I'll try that in a follow up.